mungegithub/features/repo-updates.go

```diff
@@ -26,8 +25,11 @@ import (
 	"k8s.io/kubernetes/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/util/yaml"
 
+	"fmt"
 	"github.com/golang/glog"
 	"github.com/spf13/cobra"
+	"k8s.io/contrib/mungegithub/github"
+	"path"
```

Comment:
as should path-- the three blocks are go std library imports, in-tree imports, out-of-tree (vendored) imports.